install/streams.csv

```diff
@@ -1374,6 +1374,7 @@ MWDS,21,B,N,50,true,false,false,2016-02-23T00:03:00Z,2020-06-16T00:00:00Z
 MWDS,21,H,N,200,true,false,true,2016-02-23T00:03:00Z,9999-01-01T00:00:00Z
 MWFS,20,B,N,50,true,false,false,2014-11-03T00:01:00Z,2020-06-16T00:00:00Z
 MWFS,20,H,N,200,true,false,true,2006-07-21T01:00:00Z,9999-01-01T00:00:00Z
```

Comment:
`LH` streams too?

Reply:
The LH streams are available for two days only (`2023.108` & `2023.109`). It stopped streaming since then. Should I still include it? @CBurton90

Reply:
@mnaguit Apologies, config was not set correctly. I've enabled the LH streams now so they should be coming in soon!

Reply:
@CBurton90 Okidoki...will open LH streams, using the same date/time used in HH